packages/app/cypress/e2e/runs.cy.ts

```diff
@@ -1,6 +1,35 @@
 import defaultMessages from '@packages/frontend-shared/src/locales/en-US.json'
 import type { SinonStub } from 'sinon'
 
+function scaffoldTestingTypeAndVisitRunsPage (testingType: 'e2e' | 'component') {
+  if (testingType === 'component') {
+    cy.scaffoldProject('component-tests')
```

Comment:
Can we use the `cypress-in-cypress` project? It has both testing types configured so we wouldn't have to create another system test project.

Reply:
will merge when CI goes green